mmdet/datasets/pipelines/formating.py

```diff
@@ -134,6 +134,28 @@ def __repr__(self):
 
 @PIPELINES.register_module
 class Collect(object):
+    """
+    Populates img_meta, which by default includes:
```

Comment:
Besides populating img_meta, it also includes other fields such as 'img', 'gt_bboxes', 'gt_labels'.